data/assets/examples/renderable/renderablenodearrow/renderablenodearrow_relativevalues.asset

```diff
@@ -0,0 +1,40 @@
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another, but where the size is specified using relative values (based on the bounding
+-- sphere of the start node). Here it points from Earth to the Moon
+--
```

Comment:
```suggestion

```

Reply:
Not sure I agree with this.  Care to motivate?

Reply:
Just thought it looked cleaner. Now with our new comment parsing we should keep the comments on the empty lines as well I suppose